content/get-started/01_get-started.en.md

```diff
@@ -13,25 +29,122 @@ If you want to install Cerebral into an existing project please...
 If you want an introduction you can keep reading and go through a small demo project. Just make sure you fulfill these requirements:
 
 * You have installed [Node](https://nodejs.org/en/) version 4 or later
-* You are okay with learning Cerebral using React and JSX. You can choose something else for your own project later though
+* You have installed [Git](https://git-scm.com/)
+* You are okay with learning Cerebral using React and JSX. You can choose a different Viewlayer (like [Inferno](http://infernojs.org//)) for your own project later though. 
 * You have a coffee or a beer, ready to learn something new
 
-Okay, lets get going. First you need to clone the tutorial repo.
 
-`git clone https://github.com/cerebral/cerebral-website-tutorial.git`
+### Installing the project
 
-Or use [Github Desktop](https://desktop.github.com/)
+Okay, lets get going. First you need to clone cerebral repo. So fire up your command line go to your drive and directory of choice and do:
 
-### Starting the project
-The tutorial runs on [react-create-app](https://facebook.github.io/react/blog/2016/07/22/create-apps-with-no-configuration.html) by Facebook. To run it, go to the project folder and install the dependencies, including Cerebral by running:
+`git clone https://github.com/cerebral/cerebral.git`
 
-`npm install`
+then do a  `cd cerebral/packages/cerebral-website-tutorial`
```

Comment:
not sure because cerebral-website/package.json in theory lists all deps:

```json
{
  "name": "cerebral-tutorial",
  "version": "0.1.0",
  "private": true,
  "author": "Philipp Knecht <fopsdev@gmail.com>",
  "contributors": [
    "Aleksey Guryanov <gurianov@gmail.com>",
    "Christian Alfoni <christianalfoni@gmail.com>"
  ],
  "devDependencies": {
    "react-scripts": "0.7.0",
    "shelljs": "^0.7.5"
  },
  "dependencies": {
    "cerebral": "^2.0.0-0",
    "cerebral-provider-http": "^1.0.0-0",
    "cerebral-router": "^1.0.0-0",
    "js-logger": "1.3.0",
    "react": "^15.3.2",
    "react-dom": "^15.3.2"
    
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test --env=jsdom",
    "eject": "react-scripts eject",
    "ch01": "cd ./parts/01 && rm -rf ../../src/* ../../public/*  && cp -R * ../../ && npm start",
    "ch02": "cd ./parts/02 && rm -rf ../../src/* ../../public/*  && cp -R * ../../ && npm start",
    "ch03": "cd ./parts/03 && rm -rf ../../src/* ../../public/*  && cp -R * ../../ && npm start",
    "ch04": "cd ./parts/04 && rm -rf ../../src/* ../../public/*  && cp -R * ../../ && npm start",
    "ch05": "cd ./parts/05 && rm -rf ../../src/* ../../public/*  && cp -R * ../../ && npm start",
    "ch06": "cd ./parts/06 && rm -rf ../../src/* ../../public/*  && cp -R * ../../ && npm start",
    "ch07": "cd ./parts/07 && rm -rf ../../src/* ../../public/*  && cp -R * ../../ && npm start",
    "ch08": "cd ./parts/08 && rm -rf ../../src/* ../../public/*  && cp -R * ../../ && npm start",
    "ch09": "cd ./parts/09 && rm -rf ../../src/* ../../public/*  && cp -R * ../../ && npm start",
    "ch10_1": "cd ./parts/10_1 && rm -rf ../../src/* ../../public/*  && cp -R * ../../ && npm start",
    "ch10_2": "cd ./parts/10_2 && rm -rf ../../src/* ../../public/*  && cp -R * ../../ && npm start",
    "ch11": "cd ./parts/11 && rm -rf ../../src/* ../../public/*  && cp -R * ../../ && npm start",
    "ch12": "cd ./parts/12 && rm -rf ../../src/* ../../public/*  && cp -R * ../../ && npm start",
    "ch13": "cd ./parts/13 && rm -rf ../../src/* ../../public/*  && cp -R * ../../ && npm start",
    "ch14": "cd ./parts/14 && rm -rf ../../src/* ../../public/*  && cp -R * ../../ && npm start",
    "ch15": "cd ./parts/15 && rm -rf ../../src/* ../../public/*  && cp -R * ../../ && npm start",
    "ch16": "cd ./parts/16 && rm -rf ../../src/* ../../public/*  && cp -R * ../../ && npm start"
  }
}
```